### ISSUE TITLE
Close Type Tooltip information

### ISSUE THREAD

**Issue by sendbitcoin** [Ride-The-Lightning/RTL#232]
on the closed channel overview the Close type has following types which doesn't really tell me exactly what's going on unless i'm a lightning expert:

COOPERATIVE_CLOSE
REMOTE_FORCE_CLOSE 
LOCAL_FORCE_CLOSE
FUNDING_CANCELED

It would be nice to get a tooltip with information in plain english for each type so the user can better understand exactly what is going on. something simple like: "you initalized a closing of the channel, your peer responded and you cooperatively closed the channel settling the balance onchain"

if there are different types of reasons for closing more detailed information could be displayed here, for example:
"...was closed because your peer didn't respong in time (timeout)"
"...was closed because peer responded incorrectly"
etc.

**Comment by ShahanaFarooqui** [comment]
Added below tooltips:
  COOPERATIVE_CLOSE = 'Channel closed cooperatively',
  LOCAL_FORCE_CLOSE = 'Channel force-closed by the local node',
  REMOTE_FORCE_CLOSE = 'Channel force-closed by the remote node',
  BREACH_CLOSE = 'Remote node attempted to broadcast a prior revoked channel state',
  FUNDING_CANCELED = 'Channel never fully opened',
  ABANDONED = 'Channel abandoned by the local node'
